core/src/main/java/org/elasticsearch/action/count/CountRequest.java

```diff
@@ -76,13 +62,16 @@
      */
     public CountRequest(String... indices) {
         super(indices);
+        searchSourceBuilder.size(0);
+        searchSourceBuilder.minScore(DEFAULT_MIN_SCORE);
+        searchSourceBuilder.terminateAfter(DEFAULT_TERMINATE_AFTER);
```

Comment:
I think setting the default here is not needed, SearchSourceBuilder has the same default value. Although the default minScore is null is SearchSourceBuilder I think we can remove the .minScore set too.